torch/distributed/distributed_c10d.py

```diff
@@ -1712,8 +1708,12 @@ def broadcast_object_list(object_list, src=0, group=None):
             Each object must be picklable. Only objects on the ``src`` rank will
             be broadcast, but each rank must provide lists of equal sizes.
         src (int): Source rank from which to broadcast ``object_list``.
-        group: (ProcessGroup, optional): The process group to work on. If None,
-            the default process group will be used. Default is ``None``.
+        group (ProcessGroup, optional): The process group to work on. If
+            ``None``, the default process group will be used. Default is
+            ``None``.
+        device (torch.device): Device to load tensors contained in the received
```

Comment:
Should we also use this argument instead of moving tensors to `current_device` when this `device` arg is specified?